Infrastructure/Bootstrapper.cs

```diff
@@ -25,7 +25,7 @@ public static void InitializeDefault(this IServiceCollection services, IConfigur
                                                         options.MinimumSameSitePolicy = SameSiteMode.None;
                                                     });
             services.Configure<WebOptions>(configuration);
-            services.Configure<AzureAdOptions>(configuration.GetSection("AzureAd"));
+//            services.Configure<AzureADOptions>(configuration.GetSection("AzureAd"));
```

Comment:
I've noticed that if we keep the line after using AzureADOptions instead of AzureAdOptions (which I proposed to remove as it's already in ASP.NET Core), this no longer works.